src/main/java/io/github/minecraftcursedlegacy/api/server/VanillaChecker.java

```diff
@@ -0,0 +1,15 @@
+package io.github.minecraftcursedlegacy.api.server;
+
+import io.github.minecraftcursedlegacy.impl.server.VanillaCheckerImpl;
+import net.minecraft.entity.player.Player;
+
+public final class VanillaChecker {
```

Comment:
Still think this should be a `ServerUtils` class

Reply:
having a class for a single method is not preferable

Reply:
false